src/components/Code/MultiLineRenderer.tsx

```diff
@@ -12,8 +12,9 @@ export const MultiLineRenderer: FunctionComponent<RendererProps> = ({
   <>
     <Flex
       align="center"
-      bg="rgba(0, 124, 253, 0.15)"
+      bg="gray.100"
```

Comment:
Curious what value you seen in making this change here. IMO it hurts the visual contrast of the code block from the document

new
<img width="905" alt="Screen Shot 2021-11-09 at 12 57 28 PM" src="https://user-images.githubusercontent.com/8749859/141009752-bcfa34ca-f7e5-4ebd-bd6d-310e4271424d.png">

old
<img width="892" alt="Screen Shot 2021-11-09 at 12 57 36 PM" src="https://user-images.githubusercontent.com/8749859/141009753-d1060190-ed90-4603-a9af-80f69d60a1f9.png">